tsdat/io/writers.py

```diff
@@ -87,7 +87,7 @@ def write(
             else:
                 to_netcdf_kwargs["unlimited_dims"] = ["time"]
 
-        dataset.to_netcdf(filepath, **to_netcdf_kwargs)  # type: ignore
+            dataset.to_netcdf(filepath, **to_netcdf_kwargs)  # type: ignore
 
```

Comment:
Would revert this indent, even though the behavior shouldn't change since "time" is always a dataset dimension.

Reply:
ahh ok